other/block-cluster-admin-from-ns/07-delete.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: kubectl delete -f good-pod.yaml
+- command: find . -name 'ca.*' -delete
+- command: find . -name 'testuser.*' -delete
```

Comment:
No need to do this. Everything gets trashed at the end of the tests. The only things we care about cleaning up are things in the cluster which could skew/impact further tests that get run.